docs/source/getting_started/active_learning.rst

```diff
@@ -215,7 +215,22 @@ For more information about how to use the classification scorer have a look here
 Object Detection
 ^^^^^^^^^^^^^^^^^^^^^
 Use this scorer when working on an object detection problem using bounding
-boxes.
+boxes. The object detection scorers require the input to be in 
+the `ObjectDetectionOutput` format.
+
+We expect the model predictions to contain
+
+- bounding boxes of shape (x0, y0, x1, y1)
+- objectness_probability for each bounding box
+- classification_probabilities for each bounding box
+
+You find more about the format here: 
+:py:class:`lightly.active_learning.utils.object_detection_output.ObjectDetectionOutput`
+
+We also provide a helper method to work with the model output format of only
```

Comment:
We also provide a helper method to work with the model output format consisting of only a probability per bounding box and the associated label.

(you had the of in there twice)